charts/ceph-csi-rbd/templates/nodeplugin-daemonset.yaml

```diff
@@ -124,6 +124,8 @@ spec:
               mountPropagation: "Bidirectional"
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
+            - name: rbd-nbd-logdir
```

Comment:
This is not only for `rbd-nbd` logs, but for other Ceph logs as well (in case enabled). Maybe call this `ceph-logdir` instead?

Reply:
@nixpanic  I mean that!
on the csi-rbdplugin I the only ceph client logs will be from rbd-nbd atleast now!?

Reply:
yes, only rbd-nbd logs at the moment, at least if there is no other manual configuration set. I think it is possible to reconfigure clients to write logs to their own files under `/var/log/ceph`, including the clients created by (go-ceph) libraries. There is just no need to br rbd-nbd specific in for generic paths like this.

Reply:
@pkalever afaik, mounting a host dir for an application to log while running is not a practice in kube/ocp land.  The pod restarts are handlded with same kube logging mechnism and fetch for almost all type of containers , this looks to be a non ideal approach

Reply:
@humblec 
Yes, I'm aware of this and there is an option to rbd-nbd push the logs to stderr (like what rook does https://github.com/rook/rook/blob/master/pkg/operator/ceph/config/defaults.go#L45).

But we are running rbd-nbd and csi-rbd plugin components of ceph-csi as part of the same container. The log formats of rbd plugin and rbd-nbd are different, hence it will not be wise to club the logs.

@nixpanic @Madhu-1 thoughts are welcome.

Thanks!

Reply:
AFAIK we don't persist the logs on the node in any of the ceph components in rook. is this required only for E2E or for upstream users also?

if this need to be the default
* This consumes the storage on the nodes
* Need steps to uninstall (removal of this directory)
* or instead of the default, we can document the steps to configure it? 

If this is only for E2E
* Update the daemonset templates in the E2E

Reply:
> AFAIK we don't persist the logs on the node in any of the ceph components in rook. is this required only for E2E or for upstream users also?

Probably they should persist the ceph server logs somewhere, without which debugging issue will not be possible. So I think in some way the logs will still be persistent somewhere. As an old school fellow, from my experience with gluster, we used to bind mount just like what I did in this PR.

> 
> if this need to be the default

Yes, this is not only for our e2e.

> 
> * This consumes the storage on the nodes

Yes but, IMHO rbd-nbd logs with default log level will not be a lot, probably a few KBs.

> * Need steps to uninstall (removal of this directory)
> * or instead of the default, we can document the steps to configure it?

sounds good, we can update the documentation about editing the sections of the template?

Thanks!

Reply:
@nixpanic I have respin changing name to `ceph-logdir`

Reply:
>sounds good, we can update the documentation about editing the sections of the template?

@pkalever sounds good.  is it something you are taking care of in this PR?

Reply:
Sure Madhu, it make sense to get it done within the same PR.
I will update here: https://github.com/ceph/ceph-csi/blob/devel/docs/rbd-nbd.md#configuration

Reply:
@Madhu-1 PTAL. Thanks!